bentoml/_internal/bento/bento.py

```diff
@@ -302,9 +302,111 @@ def save(
 
         return self
 
-    def export(self, path: str):
-        out_fs = fs.open_fs(path, create=True, writeable=True)
+    Format: TypeAlias = t.Literal["bento", "gz", "xz", "bz2", "tar", "zip", "folder"]
```

Comment:
The CI is failing because of missing import in this file:
```
E0602:undefined-variable [305:12] Bento: Undefined variable 'TypeAlias'
13
E0602:undefined-variable [306:17] Bento: Undefined variable 'TypeAlias'
14
E0602:undefined-variable [307:18] Bento: Undefined variable 're'
15
E0602:undefined-variable [340:27] Bento.export: Undefined variable 'hasSchemeRe'
```